proc_IR.py

```diff
@@ -168,24 +164,24 @@ def as_scan_nbr(d):
     fl.plot(s)
     x = s_signal.fromaxis('vd')
     f = fitdata(s_signal)
-    error = fitdata(s_forerror)
+    #error = fitdata(s_forerror)
     M0,Mi,R1,vd,W = symbols("M_0 M_inf R_1 vd W", real=True)
     if IR:
-        error.functional_form = Mi + (M0-Mi)*s_exp(-vd*R1)
+        #error.functional_form = Mi + (M0-Mi)*s_exp(-vd*R1)
         f.functional_form = Mi + (M0-Mi)*s_exp(-vd*R1)
     else:
-        error.functional_form = Mi*(1-(2-s_exp(-W*R1))*s_exp(-vd*R1))
+        #error.functional_form = Mi*(1-(2-s_exp(-W*R1))*s_exp(-vd*R1))
         f.functional_form = Mi*(1-(2-s_exp(-W*R1))*s_exp(-vd*R1))
-    error.fit()
+    #error.fit()
     f.fit()
     logger.info(strm("output:",f.output()))
     logger.info(strm("latex:",f.latex()))
     T1 = 1./f.output('R_1')
     fl.next('fit')
     fl.plot(s_signal,'o', label='actual data')
-    fl.plot(s_signal.imag,'o',label='actual imaginary')
+    #fl.plot(s_signal.imag,'o',label='actual imaginary')
```

Comment:
what's wrong with this?

Reply:
I forgot to uncomment I thought by commenting it out it might look better but it doesn't change anything. I will uncomment my bad!